source/stylesheets/refills/unstyled/_tooltip.scss

```diff
@@ -0,0 +1,36 @@
+.tooltip-item {  
+  display: inline-block;
+  position: relative;
+  text-align: center;
+
+  &:focus,
+  &:hover .tooltip {
```

Comment:
Is there a reason that the `:focus` selector doesn’t include `.tooltip` like `:hover` does?